### DISCUSSION TITLE
Começando com Angular - criando um projeto

### DISCUSSION THREAD

**Discussion by BrunoSts94** [balta-io/forum#510] [Tira Dúvidas] [answered]
Bom dia, primeiramente obrigado pela oportunidade de acesso aos cursos. Estou tendo um erro quando efetuo o comando ng --version, no curso e aula citados no titulo, se puderem ajudar agradeço muito. Segue anexo:
![Captura de tela 2024-03-06 085447](https://github.com/balta-io/forum/assets/133825655/f69c01bc-43a3-404a-be87-b6722fb5c5ba)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @BrunoSts94, como vai?

Você está utilizando as mesmas versões do NPM, Node e Angular do curso? Elas precisam ser iguais!

**Reply by BrunoSts94** [reply]
Há sim, pode ser isso, estou utilizando o q há de mais atual nas tecnologias. Vou verificar. Obrigado.
